composer.json

```diff
@@ -11,7 +11,7 @@
         "drupal/pathauto": "^1.6",
         "drupal/search_api": "^1.29",
         "drupal/search_api_autocomplete": "^1.3",
-        "drupal/search_api_location": "1.x-dev#cf3e546770099b1f136c020ade79a650d4178ad6",
+        "drupal/search_api_location": "1.0-alpha4",
```

Comment:
`1.0-alpha4` works as version constraint but how about `1.0.0-alpha4`?

Reply:
Yep that should work, I've updated composer to use this and will see if tests are ok again

Reply:
Thanks.